tjpcov/wigner_transform.py

```diff
@@ -230,7 +230,7 @@ def projected_covariance(
                 ell_cl=ell_cl,
                 s1_s2=s1_s2,
                 taper=taper,
-                **kwargs
+                **kwargs,
```

Comment:
This comma here is making me nervous

Reply:
This was black formatter adding this.  Should be OK because it's a multi line argument signature.... ?